presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -1066,7 +1066,7 @@ protected Object visitRow(Row node, Object context)
                 return new Row(toExpressions(values, parameterTypes));
             }
             else {
-                BlockBuilder blockBuilder =  new InterleavedBlockBuilder(parameterTypes, new BlockBuilderStatus(), cardinality);
```

Comment:
Removed in efebbd90a82739fe067f2d86a92b1bde998086fd and will add back in a later commit.